### DISCUSSION TITLE
DagsterApiServer Run execution process was terminated by signal 7 error

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17695] [Q&A]
Hi dagster support, running into this issue
`DagsterApiServer Run execution process was terminated by signal 7 (SIGBUS).`
And checking the logs:
> Sorry, logs can't be displayed.
> Please report this error to the Dagster team via GitHub or Slack. Refresh the page to try again.
> Cannot read properties of undefined (reading 'state')
This is using mem_io_manager and in_process_executor

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699031379655149?thread_ts=1699031379.655149&cid=C01U954MEER).*

**Comment by alangenfeld** [comment]
> signal.SIGBUS[¶](https://docs.python.org/3/library/signal.html#signal.SIGBUS)
> Bus error (bad memory access).

A memory error like this is likely caused by a compiled extension brought in by a library that you are using.

**Comment by mlarose** [comment]
There are also reported instances when running on Docker Desktop for Mac where using VirtioFS instead of gGRPC FUSE for file sharing causes this issue.
